presidio-anonymizer/app.py

```diff
@@ -31,28 +40,26 @@ def anonymize():
             content = request.get_json()
             if not content:
                 return ErrorResponse("Invalid request json").to_json(), 400
-            engine = AnonymizerEngine()
-            data = AnonymizerRequest(content, engine.builtin_anonymizers)
-            text = engine.anonymize(data)
+
+            data = AnonymizerRequest(content, self.engine.builtin_anonymizers)
+            text = self.engine.anonymize(data)
             return jsonify(result=text)
 
         @self.app.route("/anonymizers", methods=["GET"])
         def anonymizers() -> Tuple[str, int]:
             """Return a list of supported anonymizers."""
-            return json.dumps(AnonymizerEngine().anonymizers()), 200
+            return json.dumps(self.engine.anonymizers()), 200
 
         @self.app.errorhandler(InvalidParamException)
         def invalid_param(err):
             self.logger.warning(
-                f"failed to anonymize text with validation error: {err.err_msg}")
+                f"failed to anonymize text with validation error: {err.err_msg}"
+            )
             return ErrorResponse(err.err_msg).to_json(), 422
 
         @self.app.errorhandler(Exception)
         def server_error(e):
-            self.logger.error(
-                "A fatal error occurred "
-                "during execution".format(e)
-            )
+            self.logger.error("A fatal error occurred " "during execution".format(e))
```

Comment:
missing e placeholder?

Reply:
Not sure I follow?

Reply:
e is not printed, not something you changed though

Reply:
Fixed